instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_deeperCerebellum.jsonld

```diff
@@ -24,13 +24,19 @@
     },
     "hasParent": [
         {
-            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellum"
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/cerebellum"
         }
     ],
     "lookupLabel": "WHSSD_v1.01_deeperCerebellum",
     "name": "deeper cerebellum",
     "ontologyIdentifier": null,
     "relationAssessment": null,
     "versionIdentifier": "v1.01",
-    "versionInnovation": null
+    "versionInnovation": null,
+    "correctedName": "cerebellum, unspecified",
+    "alternativeName": [
+        "Cb-u",
+        "deeper cerebellum"
```

Comment:
The region is called "deeper cerebellum", doesn't need it as synonym as well. Maybe the same for some other instances as well? Could you run a check in your script to check for duplicates in the "alternative name" compared to "name" + "corrected name"? Then kick out duplicates from the alternative names?